compiler/z/codegen/OMRCodeGenPhase.hpp

```diff
@@ -51,12 +51,15 @@ class OMR_EXTENSIBLE CodeGenPhase : public OMR::CodeGenPhase
    public:
    static void performMarkLoadAsZeroOrSignExtensionPhase(TR::CodeGenerator * cg, TR::CodeGenPhase *);
    static void performSetBranchOnCountFlagPhase(TR::CodeGenerator * cg, TR::CodeGenPhase *);
+   static void performPreRAPeepholePhase(TR::CodeGenerator * cg, TR::CodeGenPhase *);
 
    // override base class implementation because new phases are being added
    static int getNumPhases();
    const char * getName();
    static const char* getName(PhaseValue phase);
 
+   // override base class to call new post RA peephole
+   static void performPeepholePhase(TR::CodeGenerator * cg, TR::CodeGenPhase *);
```

Comment:
What does this comment mean exactly? Also can we move up this declaration up with the others?

Reply:
80d2864